### ISSUE TITLE
Dark theme

### ISSUE THREAD

**Issue by KolyaKorruptis** [sylpheed-mail/sylpheed#5]
It would be really nice if Sylpheed supported a dark theme. With zero knowledge of GTK2 themes I managed to get a dark theme somewhat working (see screenshot), based on [Dark Olympic](https://www.gnome-look.org/p/1302313). So I know it's possible, but optimally the app should support OS theme preferences and theme switching.

![image](https://user-images.githubusercontent.com/4076947/221990758-5b4e6ec5-538d-44a6-a63a-c3a417c0e5e5.png)

**Comment by ricemark20** [comment]
I would also like a dark theme.  How did you get that to work?

**Comment by KolyaKorruptis** [comment]
I replaced the gtkrc file in Sylpheed\share\themes\MS-Windows\gtk-2.0\ with one from the linked GTK2.0 theme and then screwed around in it for a while as most of the downloaded theme was irrelevant to Sylpheed. Note that my dark theme never completely worked, eg dropdowns were unreadable white on white text.

With some CSS knowledge and knowing which items refer to what in the UI (or possibly using a dedicated tool for creating GTK2.0 themes) this really shouldn't be too hard to solve, but I don't have that time on my hands. 

For what it's worth I'm attaching my broken theming attempt.
[gtkrc.zip](https://github.com/sylpheed-mail/sylpheed/files/11143074/gtkrc.zip)

**Comment by gryf** [comment]
Dark theme can be achieved by providing right theme for GTK2. Under Linux it's very simple, as installing dependent engine and a theme, than switching them using gtk-theme-switch, or manually by editing `~/.gtkrc-2.0`.

**Comment by OOpeExtactOR** [comment]
Hello Sylpheed Developers,

Hoping this message finds you well, I wanted to express my appreciation for your dedication to developing Sylpheed. The email client has proven to be an invaluable tool for efficiently managing my emails.
Noticing the open issue (#5) regarding theme support, my interest in a dark theme option for Sylpheed has grown. Such a feature would be a significant addition, especially for users like myself who often work in low-light environments or prefer a darker interface to reduce eye strain. So I voice my support for the request made by KolyaKorruptis.
Understanding the time and effort required to develop new features, gratitude is extended for your continuous work on improving Sylpheed. The community would greatly appreciate the consideration of a dark theme in future updates.

Thank you for your time and dedication to maintaining and enhancing Sylpheed.
BR OOpeExtactOR

**Comment by ant-222** [comment]
I think this is a *GTK2* issue rather than a *Sylpheed* issue, but the *FAQ* section in the *Sylpheed* manual might include a quick how-to about importing or writing a custom theme, *if* it would be easier to follow than the official *GTK2* docs.

**Comment by KolyaKorruptis** [comment]
@ant-222 This feature request is for Sylpheed to have a UI for theme selection as well as OS theme support (auto switch dark/bright mode) like most modern apps have. I acknowledged the fact that you can manually style Sylpheed in the first post, but this is not what this issue is about.

**Comment by ant-222** [comment]
Yes, I understand. But this functionality you request is largely dependent not on *Sylpheed* itself, but on the obsolete *GTK2* library it relies on for *UI*. My point is the your request *may* be impossible to fulfil unless *Sylpheed* is migrated to the current *GTK4*, which *may* be a hard work and will probably break compatibility with older OSes, such as *Windows XP* that I am using on my PC.

**Comment by KolyaKorruptis** [comment]
@ant-222 I don't see why Sylpheed couldn't get the OS preference and start up with an appropriate dark/bright GTK2 theme? Similarly the UI could restart Sylpheed with a different GTK2 theme.

**Comment by janagyjr** [comment]
A dark theme would be nice to have, regardless of how it's done. I don't have the time to manually create one myself so having a theme-switcher would be nice.
